Removing OS Compute Hypervisors list request

mah88 opened the pull request.

mah88 committed:
Removing OS Compute Hypervisors list request

gc4rella reviewed (changes requested):
There is no need to keep that method when the call is not executed. Maybe good to remove it completely.

mah88 committed:
Update CoreModule.java

mah88 committed:
Removing ComputeNodeMap

removing computeNodeMap since it depends on the OS compute hupervisors list request

mah88 committed:
Update CoreModule.java
